Dockerfile.gcc

```diff
@@ -123,9 +120,9 @@ RUN --mount=type=cache,target=/root/.cache/ccache,id=zfs-ccache-${TARGETARCH} \
     ./scripts/make_gitrev.sh
 
 RUN --mount=type=cache,target=/root/.cache/ccache,id=zfs-ccache-${TARGETARCH} \
-    make -C module -j$(nproc) && \
+    make CC="ccache gcc" -C module -j$(nproc) && \
```

Comment:
`CC=${CC}` , like you did above....

Reply:
@rene @christoph-zededa it seems ENV is not inherited in some stages, this is expected docker behaviour. I think I lost it while merging stuff.